.dassie/config/environments/development.rb

```diff
@@ -29,6 +29,8 @@
     config.cache_store = :null_store
   end
 
+  config.web_console.whitelisted_ips = ['172.18.0.0/16', '172.27.0.0/16', '0.0.0.0/0']
+
```

Comment:
this line makes the webconsole work in docker

Reply:
let's get this in now as a separate PR?